### PR TITLE
Cleans up Hydroponics content.

### PR DESCRIPTION
**Pull request by SweptWasTaken**
## About the PR 

Hydroponics content is very dirty and disorganized. I am cleaning it up.

= Putting stuff where it should be
= Adding stuff like inhands and updating some sprites
= Getting YAML and rsis up to standard

### CODE REVIEW

**Comment by SweptWasTaken** [comment]
This is ready.

File: Resources/Textures/Clothing/Shoes/Boots/jackboots.rsi/desktop.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
+jackboots.png=@jackboots.png,0
```
Comment (PaulRitter) [inline Resources/Textures/Clothing/Shoes/Boots/jackboots.rsi/desktop.ini:2]: google drive moment

**Review by PaulRitter — CHANGES_REQUESTED** [review]

**Comment by SweptWasTaken** [comment]
Fixed the one review and updated it from the last hydroponics PR.

**Review by PaulRitter — APPROVED** [review]

**Comment by metalgearsloth** [comment]
Do not research

**Comment by Visne** [comment]
Pretty sure this messed up the spade texture.

**Pull request merged**